### PR TITLE
[StructuralMechanicsApplication] Fixing relative paths and removing unused test

### LINKED ISSUE #3866: IsotropicDamageSimoJu test failed in cl_test of structural_mechamics_application
the test IsotropicDamageSimoJu can not get constitutive law, even though i make the material document. please check it. 
the constitutive law should be IsotropicDamageSimoJuPlaneStress2DLaw.

### PR DESCRIPTION
**Pull request by loumalouomega**
Fixes #3866 and removes the errors due to relative paths

### CODE REVIEW

File: applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py [hunk 1]
```diff
@@ -235,7 +237,8 @@ def _create_reference_solution(current_model):
     }
     """)
 
-    out_parameters["output_file_name"].SetString("cl_test/test_perfect_plasticity_implementation_verification_reference.json")
+    input_filename = os.path.dirname(os.path.realpath(__file__))
+    out_parameters["output_file_name"].SetString(input_filename + "/cl_test/test_perfect_plasticity_implementation_verification_reference.json")
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py]: could you please use the same as in the core for this?
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/test_gid_io.py#L11-L12

I think the current version won't work on Win

**Comment by philbucher** [comment]
I just checked, this test is 1.9MB !!!
IMO this is way too large, the repository is large enough already

FYI @KratosMultiphysics/structural-mechanics , please keep the size of the tests small

**Comment by loumalouomega** [comment]
> I just checked, this test is 1.9MB !!!
> IMO this is way too large, the repository is large enough already
> 
> FYI @KratosMultiphysics/structural-mechanics , please keep the size of the tests small

I am removing it...

File: applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py]: Done

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/tests/test_perfect_plasticity_implementation_verification.py]: But is basically the same with copy/paste

**Comment by loumalouomega** [comment]
Done

**Comment by philbucher** [comment]
> > I just checked, this test is 1.9MB !!!
> > IMO this is way too large, the repository is large enough already
> > FYI @KratosMultiphysics/structural-mechanics , please keep the size of the tests small
> 
> I am removing it...

I know
:+1:

**Review by philbucher — APPROVED** [review]
:+1:

**Pull request merged**